lighthouse-core/test/audits/treemap-data-test.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const TreemapData_ = require('../../audits/treemap-data.js');
+const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
+const {loadSourceMapFixture, makeParamsOptional} = require('../test-utils.js');
+
+/* eslint-env jest */
+
+const TreemapData = {
+  audit: makeParamsOptional(TreemapData_.audit),
+  prepareTreemapNodes: makeParamsOptional(TreemapData_.prepareTreemapNodes),
+};
+
+/**
+ * @param {string} name
+ */
+function load(name) {
+  const data = loadSourceMapFixture(name);
+  if (!data.usage) throw new Error('exepcted usage');
```

Comment:
just move this check into `loadSourceMapFixture`? The majority of that function is loading the usage...if some hypothetical test in the future just wants a source map, it could just do the `readFileSync` call without the utility

Reply:
That doesn't seem more onerous than tests that need a pair of trace and devtoolsLogs :) but if keeping, maybe add a comment to these, then? It took me a while to realize the identity `data` spread wasn't for a shallow clone or something and was for the type checking.

Reply:
decided to just export two different functions to avoid the type issue